src/views/addManager/addManager.js

```diff
@@ -0,0 +1,280 @@
+import React, { useContext, useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { Form, Field, Formik } from "formik";
+import * as Yup from "yup";
+import * as axios from "axios";
+import { AddProperty } from '../addProperty/addProperty';
+import { SearchPanel, SearchPanelVariant } from "react-search-panel";
+import UserContext from "../../UserContext";
+import Button from "../../components/Button";
+import Modal from '../../components/Modal';
+import Toast from '../../utils/toast';
+import useMountEffect from '../../utils/useMountEffect';
+import './addManager.scss';
+import RoleEnum from '../../Enums/RoleEnum';
+
+const validationSchema = Yup.object().shape({
+  firstName: Yup.string()
+    .max(255, "Must be shorter than 255 Characters")
+    .required("Must enter a First Name"),
+  lastName: Yup.string()
+    .max(255, "Must be shorter than 255 Characters")
+    .required("Must enter a Last Name"),
+  phone: Yup.string()
+    .min(
+      5,
+      "*Number must contain at least 5 digits to be a valid phone/text number"
+    )
+    .max(20, "*Numbers can't be longer than 20 digits")
+    .required("*a valid phone number is required"),
+  email: Yup.string()
+    .email("Must be a valid email address")
+    .max(255, "Must be shorter than 255")
+    .required("Must enter an email"),
+});
+
+const makeAuthHeaders = ({ user }) => ({ headers: { 'Authorization': `Bearer ${user.accessJwt}` } });
+
+export const AddManager = () => {
+  const context = useContext(UserContext);
+  const [propertySearchText, setPropertySearchText] = useState("");
+  const [propertySelection, setPropertySelection] = useState([]);
+  const [propertyOptions, setPropertyOptions] = useState([]);
+  const [propertySearchResults, setPropertySearchResults] = useState([]);
+  const [showAddProperty, setShowAddProperty] = useState(false);
+
+  useMountEffect(() => getProperties());
+
+  useEffect(() => {
+    let choices = propertyOptions.filter(
+      p => p.description.toLowerCase().includes(propertySearchText.toLowerCase()));
+    setPropertySearchResults(choices);
+  }, [propertySearchText, propertyOptions]);
+
+  const getProperties = () => {
+    axios.get("/api/properties", makeAuthHeaders(context))
+      .then(({ data }) => {
+        let properties = data.properties && data.properties.length > 0
+          ? data.properties.map(property => {
+            return {
+              key: property.id,
+              description: `${property.name}, ${property.address}`
+            };
+          })
+          : data.properties;
+        setPropertyOptions(properties);
+        setPropertySearchResults(properties);
+        setShowAddProperty(false);
+      });
+  };
```

Comment:
Did you want to provide any error handling here?  (.catch/response==200, etc.)

Reply:
@ccbass - I think Hugh may have addressed this concern in the latest commits to the PR. Might be worth a re-review.

Reply:
I added some error checking here. Thanks for the heads-up @ccbass !